src/components/ButtonDropdown/ButtonDropdown.tsx

```diff
@@ -5,6 +5,7 @@ import styles from './ButtonDropdown.module.css';
 import { DropdownMenu } from '../..';
 import type { ClickableStyleProps } from '../ClickableStyle';
 
+// TODO: This component is deprecated and will be replaced by the Menu component
```

Comment:
Can you put this deprecated info in the component docstring instead? You can look at the `Banner` component for an example of the current pattern. It has a message at the beginning and a jsdoc tag at the end.
- https://github.com/chanzuckerberg/edu-design-system/blob/main/src/components/Banner/Banner.tsx#L94
- https://github.com/chanzuckerberg/edu-design-system/blob/main/src/components/Banner/Banner.tsx#L113